src/components/organisms/Form/hooks/useForm.ts

```diff
@@ -3,6 +3,7 @@ import { useState, useCallback, useEffect, useMemo } from 'react';
 type TValue = any;
 export type TValues = Record<string, TValue>;
 export type TErrors = Record<string, string | undefined>;
+export type TTouched = Record<string, boolean>;
```

Comment:
Following your PR [on renaming these types](#196), should we call this `TFormTouched` then?

Reply:
Since it might be considered breaking change, I'm gonna hold off with renaming for now

Reply:
addresses here https://github.com/zopaUK/react-components/pull/207